packages/cozy-konnector-libs/package.json

```diff
@@ -47,6 +47,7 @@
     "uuid": "^3.1.0"
   },
   "scripts": {
+    "build": "babel src --out-dir dist",
```

Comment:
Why do you duplicate `transpile` script?

Reply:
This is an alias because I prefer yarn build.

I should probably do : "build" : `"npm run transpile"`